pages/Documents/ConversationalAI/Third-PartyBots/public-api.md

```diff
@@ -39,21 +39,30 @@ To identify the Third-Party Bots API endpoint user, first get domain information
 | z2.bot-connectors.liveperson.net | https://bot-platform-api.emea.fs.liveperson.com/ |
 | z3.bot-connectors.liveperson.net | https://bot-platform-api.apac.fs.liveperson.com/ |
 
-#### Step 2. Get Bearer Token
+#### Step 2. Create a Public API Bot User
 
-To use our Public API you must perform a login request to Third-Party Bots API domain which will in response send you a Third-Party Bots bearer token. There are two ways in which you can perform the login.
+In order to use Public API you must create a dedicated Public API bot agent/user via LivePerson User Management UI. You must not use this dedicated Public API agent/user user in any other LivePerson business process. For example, Using the same Public API Bot Agent to create a Third-Party Bot or using this Bot Agent user to call other LivePerson APIs. The reason for this requirement is to ensure Public API bot agent's session is not invalidated by any other service/business process.
+
+Figure 2.1 shows a simple scenario below where a dedicated Public API User is first making a login call and then sending "Done" message to and ongoing conversation of a Customer with a Third-Party Bot.
+
+<img class="fancyimage" style="width:900px" src="img/tpbPublicApi/usage-diagram-public-api-message.png">
+Figure 2.1 Public API simple flow of sending messages command
+
+#### Step 3. Get Bearer Token
+
+To use our Public API you must perform a login request to Third-Party Bots API domain which will in response send you a Third-Party Bots bearer token. Moreover, You do not need to perform Login Request multiple times. One successful login call will send a bearer that can be used with multiple commands of Public API. There are two ways in which you can perform the login.
```

Comment:
Hard to comment here on a specific part. I always find it useful to use a sensible max line length in the markdown as well even if it's not enforce it the project. Side effect would be that i can comment easier on what I think should be changed.

"Moreover, You do not need to perform Login Request multiple times"
"Moreover, you do not need to perform login request multiple times"